openwisp_notifications/handlers.py

```diff
@@ -205,6 +204,7 @@ def send_email_notification(sender, instance, created, **kwargs):
             'email_id': instance.recipient.email,
         },
     )
+    EMAIL_BATCH_INTERVAL = app_settings.EMAIL_BATCH_INTERVAL
```

Comment:
This is executed at runtime, therefore we can override `app_settings.EMAIL_BATCH_INTERVAL` during tests with `mock.patch`.